Source/Core/UICommon/UICommon.cpp

```diff
@@ -150,9 +150,13 @@ void SetUserDirectory(const std::string& custom_path)
 		// We are on a non-Apple and non-Android POSIX system, let's respect XDG basedir.
 		// The only case we don't is when there is an existing ~/.dolphin-emu directory.
 		// See http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html
-
 		user_path = home_path + "." DOLPHIN_DATA_DIR DIR_SEP;
-		if (!File::Exists(user_path))
+		std::string exe_path = File::GetExeDirectory();
+		if (File::Exists(exe_path + DIR_SEP "portable.txt"))
+		{
+			user_path = exe_path + DIR_SEP "PortableConfig" DIR_SEP;
```

Comment:
Windows uses `USERDATA_DIR` (`"User"`) instead of `"PortableConfig"`. Should that be used here too for consistency?